qiskit/pulse/channels.py

```diff
@@ -13,89 +13,107 @@
 # that they have been altered from the originals.
 
 """
-Channels.
+This module defines Pulse Channels. Channels include:
+
+  - transmit channels, which should subclass``PulseChannel``
+  - receive channels, such as ``AcquireChannel``
+  - non signal "channels" such as SnapshotChannel, MemorySlot and RegisterChannel, which relate
+    more closely to the classical control hardware.
```

Comment:
```suggestion
    other device properties.
```

Reply:
What is another way to say this that is more specific? especially since `properties` refers to measured values like `BackendProperties.t1`